src/modules/app/containers/__tests__/__snapshots__/RouteContentSwitcher.spec.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RouteContentSwitcher renderFailureLoadingComponents should render a FailureSwitcher if a payload contains an error 1`] = `
+<Translate(FailureSwitcher)
```

Comment:
Probably should dive() one layer deeper

Reply:
Why? we want to render the FailureSwitcher component with the translate function, but we don't care how and what exactly is rendered. This test should test the RouteContentSwitcher and not the FailureSwitcher, so imo this is enough.